### ISSUE TITLE
3.1.1 Jar distribution missing (search.maven.org)

### ISSUE THREAD

**Issue by aquator** [robotframework/robotframework#3053]
Hey,

It seems that the fixes are not released to Maven Central. Can you please share them there as well?

Thanks!

**Comment by pekkaklarck** [comment]
This is true. I'm upgrading my laptop and GPG keys needed for uploading the JAR are on the old hard drive. I may not have time to upload the JAR before RobCon but latest next week.

**Comment by pekkaklarck** [comment]
@Hi-Fi promised to help with this. Hopefully the JAR is up soon!
